astropy/units/structured.py

```diff
@@ -360,12 +360,39 @@ def is_equivalent(self, other, equivalencies=[]):
 
         return True
 
-    def _get_converter(self, other, equivalencies=[]):
+    def get_converter(self, other, equivalencies=[]):
+        """
+        Return the conversion function to convert values from ``self``
```

Comment:
Comments on the docstring like above. Can we be blunt and just do `get_converter.__doc__ = UnitBase.get_converter.__doc__`?